core/src/main/java/org/apache/accumulo/testing/core/performance/Result.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.accumulo.testing.core.performance;
+
+public class Result {
+
+  public final String id;
+  public final Number data;
+  public final Stats stats;
+  public final String description;
+  public final Purpose purpose;
+
+  public enum Purpose {
+    INFORMATIONAL, COMPARISON
```

Comment:
It might nice to add some comments describing the difference between informational and comparison.